lib/NormalModuleReplacementPlugin.js

```diff
@@ -27,7 +27,7 @@ class NormalModuleReplacementPlugin {
 				}
 				return callback(null, result);
 			});
-			nmf.plugin("after-resolve", (result, callback) => {
+			nmf.hooks.afterResolve.tapAsync("NormalModuleReplacementPlugin", (result, callback) => {
```

Comment:
Use `tap` instead of `tapAsync` and remove the `callback`. Return the result instead.